xbmc/Application.h

```diff
@@ -306,42 +311,21 @@ class CApplication : public CXBApplicationEx, public IPlayerCallback, public IMs
 
   int GlobalIdleTime();
 
-  void EnablePlatformDirectories(bool enable=true)
-  {
-    m_bPlatformDirectories = enable;
-  }
+  void EnablePlatformDirectories(bool enable = true) { m_bPlatformDirectories = enable; }
 
-  bool PlatformDirectoriesEnabled()
-  {
-    return m_bPlatformDirectories;
-  }
+  bool PlatformDirectoriesEnabled() { return m_bPlatformDirectories; }
```

Comment:
the way it was is much better. more than a single expression on a single line makes debugging harder

Reply:
In the general case yes, but how exactly does that apply here? Function has only one statement anyway so as I see it neither breakpoints nor stepping should be affected in any way.

Reply:
Considering it as a general rule make it easier. Contributers are inclined to keep the style when they do changes and this my become something like

```
bool PlatformDirectoriesEnabled() { return m_bPlatformDirectories && SomethingElse(); }
```

Reply:
OK, fair point.

About the example you wrote - how should you format it then?

Reply:
Forcing multi-line is possible afaik, either way is fine by me

Reply:
@pkerling I would write it like this. when debuggong you don't have to step into SomethingElse()

```
bool PlatformDirectoriesEnabled()
{
  if (!SomethingElse())
    return false;
  return m_bPlatformDirectories
}
```

Reply:
updated to never allow single line functions now.